frontend/src/components/SubmissionHistory.vue

```diff
@@ -19,9 +19,8 @@
           mobile-breakpoint="md"
           fixed-header
           class="elevation-4 my-4"
-          disable-pagination
-          hide-default-footer
-          :sort-by="['submissionDate']"
+          :items-per-page="10"
```

Comment:
Looks good to me - I am going to wait on approving this PR until we get business confirmation that this should be in the next build